extension.meta.xml

```diff
@@ -22,8 +22,12 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="2.0.0" date="TBA" min="4.0.0" max="4.x.x" php-min="5.6.x" php-max="7.x.x">

```

Comment:
set today's date, set the min to "2.3.3" and max to "2.x.x" Thanks !

Reply:
Done.